html-forms/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form action="" method="post">
+    <div>
+      <label for="email">Email
+        <br>
+        <input type="text" name="email" id="email">
+      </label>
+      <br>
+      <label for="coding-reasons">Why is coding cool?
+        <br>
+        <select name="coding-reasons" id="coding-reasons">
+          <option value="select">Select an option</option>
+          <option value="interesting">Because it's interesting</option>
+          <option value="fun">Because it's fun</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </label>
+    </div>
+    <br>
+    <div>
+      Favorite Language (choose one)
+    </div>
+    <br>
+    <div>
+      <input type="radio" name="language" value="html" id="html">
+      <label for="html">HTML</label>
+      <input type="radio" name="language" value="css" id="css">
+      <label for="css">CSS</label>
+      <input type="radio" name="language" value="javascript" id="javascript">
+      <label for="javascript">JavaScript</label>
+    </div>
+    <br>
+    <div>
+      Would you like to sign up for more information?
+    </div>
+    <br>
+    <div>
+      <input type="checkbox" name="sign-up" value="yes" id="sign-up">
+      <label for="sign-up">Sign me up!</label>
+    </div>
+    <div>
+      <label for="">Comments</label>
```

Comment:
This `label`'s `for` attribute needs a value matching the `id` attribute on its paired form control element.